integration/test/src/local/Tests.scala

```diff
@@ -10,4 +10,4 @@ object UpickleTests extends mill.integration.UpickleTests(fork = false)
 object PlayJsonTests extends mill.integration.PlayJsonTests(fork = false)
 object CaffeineTests extends mill.integration.CaffeineTests(fork = false)
 object DocAnnotationsTests extends mill.integration.DocAnnotationsTests(fork = false)
-object ScriptsInvalidationTests extends mill.integration.ScriptsInvalidationTests(fork = true)
+object ScriptsInvalidationTests extends mill.integration.ScriptsInvalidationTests(fork = false)
```

Comment:
We typically also have an instance with `fork = true` in package `mill.integration.forked`.

Reply:
Now I understand why I was breaking the convention of the test being `fork = true` in the wrong place! :D
Done by the way :white